studio/pages/new-with-subscription.tsx

```diff
@@ -0,0 +1,127 @@
+import { useCallback, useEffect, useState } from 'react'
+import { observer } from 'mobx-react-lite'
+
+import { API_URL, STRIPE_PUBLIC_KEY } from 'lib/constants'
+import { useStore } from 'hooks'
+import { post } from 'lib/common/fetch'
+import { WizardLayout } from 'components/layouts'
+import { NextPageWithLayout } from 'types'
+import { Elements } from '@stripe/react-stripe-js'
+import { loadStripe } from '@stripe/stripe-js'
+import { useIsHCaptchaLoaded } from 'stores/hcaptcha-loaded-store'
+import HCaptcha from '@hcaptcha/react-hcaptcha'
+import { NewOrgForm } from 'components/interfaces/Organization'
+import { useTheme } from 'common'
+
+const stripePromise = loadStripe(STRIPE_PUBLIC_KEY)
+
+/**
+ * No org selected yet, create a new one
+ */
+const Wizard: NextPageWithLayout = () => {
+  const { ui } = useStore()
+
+  const [intent, setIntent] = useState<any>()
+  const captchaLoaded = useIsHCaptchaLoaded()
+
+  const [captchaToken, setCaptchaToken] = useState<string | null>(null)
+  const [captchaRef, setCaptchaRef] = useState<HCaptcha | null>(null)
+
+  const { isDarkMode } = useTheme()
+
+  // TODO make sure to redirect away from this page if feature toggle is not true
```

Comment:
Need help here, what's the best way to navigate away if the featureToggle is false?